crates/bindings-csharp/Runtime/Internal/FFI.cs

```diff
@@ -19,7 +19,7 @@ public readonly record struct BytesSource(uint Handle)
 public readonly record struct BytesSink(uint Handle) { }
 
 #pragma warning disable IDE1006 // Naming Styles - Not applicable to FFI stuff.
-internal static partial class FFI
+public static partial class FFI
```

Comment:
Is this just for Errno? Can we extract it like BytesSource above if so? I'd prefer not to expose APIs we can easily keep hidden.